.github/workflows/build.yml

```diff
@@ -106,27 +103,20 @@ jobs:
         with:
           repository: daeuniverse/dae-wing
           path: dae-wing
-          ref: 'd986b18e10a7bfb90da1c15094b97e9386c459ef'
```

Comment:
Why remove version pinning?

Reply:
Considering the sqlite problem fix is merged, this is not required anymore